.github/workflows/check_pr_fixlib.yml

```diff
@@ -62,5 +62,5 @@ jobs:
         uses: pypa/gh-action-pypi-publish@release/v1
         with:
           user: __token__
-          password: ${{ secrets.PYPI_RESOTOLIB }}
-          packages_dir: ./resotolib/dist/
+          password: ${{ secrets.PYPI_FIXLIB }}
```

Comment:
Python package name is `fixinventorylib` (fixlib was taken) secret is `PYPI_FIXINVENTORYLIB`.